jetty-server/src/main/java/org/eclipse/jetty/server/HttpInput.java

```diff
@@ -139,9 +143,23 @@ public int read(byte[] b, int off, int len) throws IOException
     {
         synchronized (_inputQ)
         {
+            long now=System.currentTimeMillis();
```

Comment:
We should really try to get rid of `System.currentTimeMillis()`, as it is subject to NTP etc., see also #121. Unfortunately here we are tied to what returned by `Request.getTimeStamp()`.